server/tests/api/test_support.py

```diff
@@ -770,3 +770,55 @@ def test_support_clear_offline_results_wrong_audit_type(
             }
         ]
     }
+
+
+def test_reopen_current_round(
+    client: FlaskClient, election_id: str, contest_ids: List[str], round_1_id: str,
+):
+    def is_round_completed(round_id: str) -> bool:
```

Comment:
When possible, I try to test Arlo using the API interface rather than looking into the db. Is there a way that the support API will show that the round has been reopened? If so, can that be used in the test? (The support user interface will probably need some way to know this anyway)

I'm fine with keeping these assertions in addition if it's important to make sure that all the relevant data was deleted (which in this case it might be).

Reply:
Makes a lot of sense! As far as I could tell, there wasn't already a way to retrieve round data using the support user API, so I updated the get-election endpoint. And I used that endpoint/data where I could in the round-reopening tests

https://github.com/votingworks/arlo/pull/1571/commits/baf5113e9b6acffec6559f1180f3e01b083a10bd